README.md

```diff
@@ -79,6 +79,43 @@ Collectors are enabled by providing a `--collector.<name>` flag.
 Collectors that are enabled by default can be disabled by providing a `--no-collector.<name>` flag.
 To enable only some specific collector(s), use `--collector.disable-defaults --collector.<name> ...`.
 
+### Include & Exclude flags
+
+A few collectors can be configured to include or exclude certain patterns using dedicated flags. The flags are mutually exclusive on collectors that support both.
```

Comment:
Maybe clarify that exclude means 'all except' while include means 'none except'

Reply:
Done :

> A few collectors can be configured to include or exclude certain patterns using dedicated flags. The exclude flags are used to indicate "all except", while the include flags are used to say "none except". Note that these flags are mutually exclusive on collectors that support both.